packages/jest-leak-detector/package.json

```diff
@@ -20,6 +20,9 @@
     "jest-get-type": "workspace:^",
     "pretty-format": "workspace:^"
   },
+  "devDependencies": {
+    "@types/node": "*"
```

Comment:
https://github.com/facebook/jest/blob/3d36b608992fc56e0aa7295429bb80dbc8774ea9/packages/jest-leak-detector/src/index.ts#L9-L11

but stripped out of the compiled file